rad/src/script.rs

```diff
@@ -211,10 +213,17 @@ fn errorify(kind: RadError) -> RadError {
 pub fn create_radon_script_from_filters_and_reducer(
     filters: &[RADFilter],
     reducer: u32,
+    active_wips: Option<&ActiveWips>,
 ) -> Result<Vec<RadonCall>, RadError> {
     let unknown_filter = |code| RadError::UnknownFilter { code };
     let unknown_reducer = |code| RadError::UnknownReducer { code };
 
+    let active_wips = if let Some(active_wips) = active_wips {
+        active_wips.clone()
+    } else {
+        current_active_wips()
+    };
```

Comment:
I think this clone can be avoided, by doing something like this:

```suggestion
    let current_wips;
    let active_wips = if let Some(active_wips) = active_wips {
        active_wips
    } else {
        current_wips = current_active_wips();
        &current_wips
    };
```